TAScheduler/views_methods.py

```diff
@@ -404,6 +405,37 @@ def courseTAAsmgt(self, active_ta, active_course):  # new remove sectionTAAsmgt
             raise RuntimeError("Instructor is already assigned to max number of course permitted")
         TAToCourse.objects.create(ta=active_ta.database, course=active_course.database)
 
+    def sectionTAAsmgt(self, active_ta, active_section):
+        if not isinstance(active_ta, TA):
+            raise TypeError("Input passed to admin.sectionTAAsmgt is not TA")
+        if not isinstance(active_section, LabObj) and not isinstance(active_section, LectureObj):
+            raise TypeError("Input passed to admin.sectionTAAsmgt is not Lab or Lecture")
+        active_section.addTA(active_ta)
+
+    def getAllCrseAsgmts(self):
+        outputdict = {}
+        if InstructorToCourse.objects.all().count() + TAToCourse.objects.all().count() == 0:
+            raise RuntimeError("No course links exist")
+        for i in InstructorToCourse.objects.all():
+            outputdict[i.course.course_id] = i.instructor.user.email_address
+        for i in TAToCourse.objects.all():
+            if i.course.course_id in outputdict:
+                outputdict[i.course.course_id] = (outputdict[i.course.course_id], i.ta.user.email_address)
+            else:
+                outputdict[i.course.course_id] = i.ta
```

Comment:
can replace with 
outputdict.setdefault(i.course.course_id, []).append(i.ta.user.email_address) 
for more consistency